static/js/util/util.js

```diff
@@ -554,3 +554,11 @@ export const mapObj = R.curry((fn, obj) => R.compose(
   R.map(fn),
   R.toPairs
 )(obj));
+
+export const wait = (millis: number): Promise<void> => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve();
+    }, millis);
```

Comment:
since we're not passing any value to `resolve` here (for the `.then` handler to grab later) could we just do

```js
return new Promise(resolve => {
  setTimeout(resolve, millis);
});
```